src/transcript-plugin.tsx

```diff
@@ -155,6 +157,24 @@ export class TranscriptPlugin implements OnMediaLoad, OnPluginSetup, OnMediaUnlo
           />
       )
     });
+    this._contribServices.upperBarManager.add({
+      label: 'Info',
+      onClick: () => {},
+      renderItem: () => (
+        <button
+          className={styles.transcriptIcon}
+          tabIndex={1}
+      />
+      ),
+    });
+  }
+
+  private _handleIconClick = (event: MouseEvent) => {
+    if (event.x === 0 && event.y === 0) {
```

Comment:
there has to be a better way to identify the events not by X & Y - try to see what is the event type ...

Reply:
onClick generates ONLY mouse event (even if onClick triggered by keyboard);
`onClick={this._handleIconClick}`
only one way to determine if button triggered by keyboard or mouse - it's check the click coordinates;
otherwise we have to use different listeners - for mouse "click", for touchscreens "touch" and for keyboard "pressed";